snforge_std/Scarb.toml

```diff
@@ -3,4 +3,4 @@ name = "snforge_std"
 version = "0.1.0"
 
 [dependencies]
-starknet = "2.3.0"
+starknet = "2.3.1"
```

Comment:
I insist that we don't bump it, we should stop bumping it in general due. Let's say we have forge 0.11.0 compatible with Scarb 2.3.0 and 2.3.1 (which will be the case). Starknet package is coupled with Scarb, so Scarb 2.3.0 can only provide starknet 2.3.0, which is incompatible with starknet 2.3.1

Reply:
ok thx for notifying me!